tests/foreman/cli/test_template.py

```diff
@@ -74,6 +78,75 @@ def test_update_template_1(self):
         self.assertEqual(len(result.stderr), 0)
         self.assertEqual(updated_name, result.stdout['name'])
 
+    def test_create_template_with_location(self):
+        """@Test: Check if Template with Location can be created
+
+        @Feature: Template - Create
+
+        @Assert: Template is created and new Location has been assigned
+
+        """
+
+        name = gen_string("alpha", 10)
```

Comment:
It is best to avoid variables that are used only once. I suggest changing the call to ``make_template`` to assign the value of ``name`` directly:

```python
new_template = make_template(
        {
                    'name': gen_string("alpha", 10),
                    'location-ids': new_loc["id"],
                }
            )
```